tests/zfs-tests/tests/functional/cli_root/zpool_expand/zpool_expand_003_neg.ksh

```diff
@@ -27,23 +27,23 @@
 
```

Comment:
Don,
Are you certain that autoexpand depends on ZED?  See the following from spa_vdev_detach():

{code}
4944         /*
4945          * If the 'autoexpand' property is set on the pool then automatically
4946          * try to expand the size of the pool. For example if the device we
4947          * just detached was smaller than the others, it may be possible to
4948          * add metaslabs (i.e. grow the pool). We need to reopen the vdev
4949          * first so that we can obtain the updated sizes of the leaf vdevs.
4950          */
4951         if (spa->spa_autoexpand) {
4952                 vdev_reopen(tvd);
4953                 vdev_expand(tvd, txg);
4954         }
4955
4956         vdev_config_dirty(tvd);
{code}

I'm confused ...

Thanks for helping me work through these tests.